src/attributes.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const _ = require('lodash');
+
+class Attributes {
+    constructor(attributes) {
+        this.classes = [];
+        this.storage = {};
+        if (attributes !== undefined) {
+            this.merge(attributes);
+        }
+    }
+
+    addClass(...classnames) {
```

Comment:
(and elsewhere)  There's no JSDoc, because the entire package is undocumented — probably caused by the "WIP" state of the very original source, https://github.com/frctl/twig

I can happily add docs, if sensible / requested.  But given that there is no existing standard for orientation, I'll probably stick to Doxygen classics (unless someone points me to something better/relevant).